mod/admin.php

```diff
@@ -1266,7 +1266,7 @@ function admin_page_site(App $a)
 		'$banner'		=> array('banner', t("Banner/Logo"), $banner, ""),
 		'$shortcut_icon'	=> array('shortcut_icon', t("Shortcut icon"), Config::get('system','shortcut_icon'),  t("Link to an icon that will be used for browsers.")),
 		'$touch_icon'		=> array('touch_icon', t("Touch icon"), Config::get('system','touch_icon'),  t("Link to an icon that will be used for tablets and mobiles.")),
-		'$info'			=> array('info', t('Additional Info'), $info, sprintf(t('For public servers: you can add additional information here that will be listed at %s/servers.'), get_server())),
+		'$info'			=> array('info', t('Additional Info'), $info, t('For public servers: you can add additional information here that will be listed at %s/servers.', System::baseUrl()), get_server()),
```

Comment:
Why had you added ```System::baseUrl())``` here?

Reply:
If you look at the translation string, there is a `%s` but no interpolation variable was provided. I just filled in the blank.

Reply:
The ```%s``` belongs to ```get_server()```.

Reply:
😅